messaging/src/main/java/org/axonframework/eventhandling/MergedTrackingToken.java

```diff
@@ -76,6 +76,38 @@ protected MergedTrackingToken(TrackingToken lowerSegmentToken, TrackingToken upp
         this.upperSegmentAdvanced = upperSegmentAdvanced;
     }
 
+    /**
+     * Indicates whether the given {@code trackingToken} represents a position that is part of a merge.
+     *
+     * @param trackingToken The token to verify
+     * @return {@code true} if the token indicates a merge
+     */
+    public static boolean isMergeInProgress(TrackingToken trackingToken) {
+        return WrappedToken.unwrap(trackingToken, MergedTrackingToken.class).isPresent();
+    }
+
+    /**
+     * Return the estimated relative token position this Segment will have after a merge operation is complete.
+     * In case no estimation can be given or no merge in progress, an {@code OptionalLong.empty()} will be returned.
+     *
+     * @return return the estimated relative position this Segment will reach after a merge operation is complete.
+     */
+    public static OptionalLong mergePosition(TrackingToken trackingToken) {
```

Comment:
This gives the upper most position when merging should be resolved, but don't we also need the lowest point of the existing tokens? 

It's those two numbers which can give us the "merging progress", quite similar to exposing the `replayPosition` and `tokenAtResetPosition`.

Reply:
Added, I am not overly pleased with the name of this method, but haven't landed on something satisfying either...

I can only think in terms of "upper most token" and "lowest token", which would suggest that `this` method would be renamed to `upperMergePosition`/`upperMergingPosition` and the other would be `lowestMergePosition`/`lowestMergingPosition`. 

What do you think?

Reply:
I believe this issue has been covered in the other discussion around sharing a regular `getPosition` method to be used for both replaying and merging, with a dedicated finish-line-position method. Hence I'll resolve this conversation.